README.md

```diff
@@ -34,3 +34,19 @@ To see how it works just simply issue at your shell prompt:
 ```
 $ jlc-kicad-tools
 ```
+
+### KiCad
+
+1. Design your board as usual
+1. Create a custom field in Eeschema (for instance LCSCStockCode) and add in the components "C" stock number from JLCPCB/LCSC
+1. Once you are satisfied with your board design, go to Eeschema and run “Tools -> Generate Bill of Materials”
+1. Go to Pcbnew and run ‘File -> Fabrication Outputs -> Footprint Position (.pos) File’.
+1. Plot gerbers as per usual for PCB manufacturing
+1. Run the script and give it the project directory
+
+```
+generate_jlc_files.py --warn-no-lcsc-partnumber --database=cpl_rotations_db.csv --verbose myprojectfolder
```

Comment:
Please don't include optional args in the example command line. If we think people should use them by default we would have made them the default :).